ui/src/components/DebugStrings.tsx

```
@@ -0,0 +1,282 @@
+    }
+
+    return (
+      <div 
+        style={{ 
+          padding: "10px", 
+          backgroundColor: "#f9f9f9", 
+          borderRadius: "4px",
+          overflow: "auto",
+          maxHeight: "300px"
+        }}
+        dangerouslySetInnerHTML={{ __html: svgElement.outerHTML }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)